package.json

```diff
@@ -65,81 +94,131 @@
-    "compile:lowlight": "rm -rf ./build; THEME=clinical-lowlight-theme webpack -p",
-    "compile:fusion": "rm -rf ./build; THEME=orion-fusion-theme webpack -p",
-    "danger": "danger ci",
-    "dependency-markdown": "node scripts/dependency-markdown-generator/DependencyMarkdownGenerator.js",
-    "docs": "node scripts/root-readme-updater/index.js",
-    "docs:peerDependencies": "node scripts/peer-dependency-generator/index.js",
-    "heroku-prebuild": "npx lerna init",
-    "heroku-postbuild": "npm install --only=dev && npm run compile:prod",
-    "jest": "jest",
-    "jest:coverage": "jest --coverage",
-    "link-parent-bin": "link-parent-bin",
-    "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "eslint --ext .js,.jsx .",
-    "lint:scss": "stylelint 'packages/**/src/**/*.scss' --syntax scss",
-    "pretest": "npm run lint",
-    "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
-    "start": "tt-serve",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-prod": "tt-serve --env.disableHotReloading -p",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
-    "test": "npm run lint && npm run jest && npm run wdio",
-    "wdio-default": "wdio",
-    "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 THEME=clinical-lowlight-theme wdio",
-    "wdio-fusion": "THEME=orion-fusion-theme wdio",
-    "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion",
-    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js"
+  "dependencies": {
+    "d3": "^5.16.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.5.0",
     "@babel/core": "^7.5.0",
     "@babel/plugin-proposal-object-rest-spread": "^7.5.0",
     "@babel/plugin-transform-object-assign": "^7.2.0",
     "@babel/plugin-transform-runtime": "^7.5.0",
-    "@babel/preset-env": "^7.5.0",
+    "@babel/polyfill": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
     "@babel/preset-react": "^7.0.0",
-    "@cerner/eslint-config-terra": "^4.0.0",
+    "@babel/runtime": "^7.10.4",
     "@cerner/browserslist-config-terra": "^3.0.0",
+    "@cerner/carbon-site-helpers": "^1.11.3",
+    "@cerner/eslint-config-terra": "^4.0.0",
+    "@cerner/svg-to-carbon": "^1.1.3",
+    "@semantic-release/git": "^9.0.0",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^24.8.0",
+    "babel-loader": "^8.1.0",
+    "babel-plugin-istanbul": "^6.0.0",
+    "babel-plugin-minify-replace": "^0.5.0",
+    "babel-preset-minify": "^0.5.1",
     "check-installed-dependencies": "^1.0.0",
+    "classnames": "^2.2.5",
+    "commander": "^5.1.0",
     "core-js": "^3.1.3",
+    "css-loader": "^3.6.0",
+    "cssnano": "^4.1.10",
     "danger": "^9.1.0",
+    "docdash": "^1.2.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "enzyme-to-json": "^3.2.2",
     "eslint": "^6.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "express": "^4.17.1",
```

Comment:
do we still need express?

Reply:
Yes, the current carbon dev-site uses express